### PR TITLE
LPS 126032

### PR DESCRIPTION
**Pull request by joseabelenda**

### CODE REVIEW

File: modules/apps/export-import/export-import-test-util/src/main/java/com/liferay/exportimport/test/util/lar/BasePortletExportImportTestCase.java [hunk 1]
```diff
@@ -364,14 +364,33 @@ protected void deleteLatestVersion(StagedModel stagedModel)
 	protected void exportImportPortlet(String portletId) throws Exception {
 		exportImportPortlet(
 			portletId, new LinkedHashMap<String, String[]>(),
-			new LinkedHashMap<String, String[]>());
+			new LinkedHashMap<String, String[]>(), true, false);
+	}
+
+	protected void exportImportPortlet(
+			String portletId, boolean staging, boolean exporting)
```
Comment (carolmariaabb) [inline modules/apps/export-import/export-import-test-util/src/main/java/com/liferay/exportimport/test/util/lar/BasePortletExportImportTestCase.java:371]: Hi @joseabelenda , how about renaming `staging` to `portletStagingInProcess` and `exporting` to  `portletExportInProcess`? 
Also, can you order it alphabetically `(String portletId, boolean portletExportInProcess, boolean portletStagingInProcess)`? `portletId` must always come first

File: modules/apps/export-import/export-import-test-util/src/main/java/com/liferay/exportimport/test/util/lar/BasePortletExportImportTestCase.java [hunk 2]
```diff
@@ -364,14 +364,33 @@ protected void deleteLatestVersion(StagedModel stagedModel)
...
 		throws Exception {
 
+		exportImportPortlet(
+			portletId, exportParameterMap, importParameterMap, true, false);
+	}
+
+	protected void exportImportPortlet(
+			String portletId, Map<String, String[]> exportParameterMap,
+			Map<String, String[]> importParameterMap, boolean staging,
+			boolean exporting)
```
Comment (carolmariaabb) [inline modules/apps/export-import/export-import-test-util/src/main/java/com/liferay/exportimport/test/util/lar/BasePortletExportImportTestCase.java:391]: @joseabelenda same as above.

File: modules/apps/export-import/export-import-test-util/src/main/java/com/liferay/exportimport/test/util/lar/BasePortletExportImportTestCase.java [hunk 3]
```diff
@@ -390,7 +409,8 @@ protected void exportImportPortlet(
 						TYPE_PUBLISH_PORTLET_LOCAL,
 					settingsMap);
 
-		ExportImportThreadLocal.setPortletStagingInProcess(true);
+		ExportImportThreadLocal.setPortletStagingInProcess(staging);
+		ExportImportThreadLocal.setPortletExportInProcess(exporting);
```
Comment (carolmariaabb) [inline modules/apps/export-import/export-import-test-util/src/main/java/com/liferay/exportimport/test/util/lar/BasePortletExportImportTestCase.java:413]: @joseabelenda can you order it alphabetically?

```
ExportImportThreadLocal.setPortletExportInProcess(..);
ExportImportThreadLocal.setPortletStagingInProcess(...);
```

File: modules/apps/export-import/export-import-test-util/src/main/java/com/liferay/exportimport/test/util/lar/BasePortletExportImportTestCase.java [hunk 4]
```diff
@@ -448,9 +468,10 @@ protected void exportImportPortlet(
 		}
 		finally {
 			ExportImportThreadLocal.setPortletStagingInProcess(false);
+			ExportImportThreadLocal.setPortletExportInProcess(false);
```
Comment (carolmariaabb) [inline modules/apps/export-import/export-import-test-util/src/main/java/com/liferay/exportimport/test/util/lar/BasePortletExportImportTestCase.java:471]: @joseabelenda same as above.

File: modules/apps/export-import/export-import-test-util/src/main/java/com/liferay/exportimport/test/util/lar/BasePortletExportImportTestCase.java [hunk 5]
```diff
@@ -464,6 +485,20 @@ protected PortletPreferences getImportedPortletPreferences(
 		return LayoutTestUtil.getPortletPreferences(importedLayout, portletId);
 	}
 
+	protected PortletPreferences getImportedPortletPreferences(
+			Map<String, String[]> preferenceMap, boolean staging,
+			boolean exporting)
```
Comment (carolmariaabb) [inline modules/apps/export-import/export-import-test-util/src/main/java/com/liferay/exportimport/test/util/lar/BasePortletExportImportTestCase.java:490]: @joseabelenda same as my first comment

File: modules/apps/export-import/export-import-test-util/src/main/java/com/liferay/exportimport/test/util/lar/BasePortletExportImportTestCase.java [hunk 6]
```diff
@@ -364,14 +364,33 @@ protected void deleteLatestVersion(StagedModel stagedModel)
 	protected void exportImportPortlet(String portletId) throws Exception {
 		exportImportPortlet(
 			portletId, new LinkedHashMap<String, String[]>(),
-			new LinkedHashMap<String, String[]>());
+			new LinkedHashMap<String, String[]>(), true, false);
```
Comment (carolmariaabb) [inline modules/apps/export-import/export-import-test-util/src/main/java/com/liferay/exportimport/test/util/lar/BasePortletExportImportTestCase.java:367]: Don't forget to change the order in these cases, i.e, `(...,false, true)`

**Review by carolmariaabb — CHANGES_REQUESTED** [review]
Hi @joseabelenda
I believe you should fix the semantic versioning for `export-import:export-import-test-util`.

File: modules/apps/export-import/export-import-test-util/src/main/java/com/liferay/exportimport/test/util/lar/BasePortletExportImportTestCase.java (hunk 1 above)
Comment (joseabelenda) [reply modules/apps/export-import/export-import-test-util/src/main/java/com/liferay/exportimport/test/util/lar/BasePortletExportImportTestCase.java]: Hi, @carolmariaabb.

I've reviewed and the variable _exporting_ isn't necessary.
I renamed _staging_ to _portletStagingInProcess_.

File: modules/apps/export-import/export-import-test-util/src/main/java/com/liferay/exportimport/test/util/lar/BasePortletExportImportTestCase.java (hunk 3 above)
Comment (joseabelenda) [reply modules/apps/export-import/export-import-test-util/src/main/java/com/liferay/exportimport/test/util/lar/BasePortletExportImportTestCase.java]: Variable `exporting` was removed.

File: modules/apps/export-import/export-import-test-util/src/main/java/com/liferay/exportimport/test/util/lar/BasePortletExportImportTestCase.java (hunk 6 above)
Comment (joseabelenda) [reply modules/apps/export-import/export-import-test-util/src/main/java/com/liferay/exportimport/test/util/lar/BasePortletExportImportTestCase.java]: Yes, sure! 🙂

**Review by carolmariaabb — APPROVED** [review]

**Comment by joseabelenda** [comment]
@carolmariaabb, grateful for your comments.
I've reviewed the code and sent again.
I hope the code is in compliance.

Thanks in advance.

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
> @carolmariaabb, grateful for your comments.
> I've reviewed the code and sent again.
> I hope the code is in compliance.
> 
> Thanks in advance.

Thank you @joseabelenda !

**Comment by carolmariaabb** [comment]
ci:forward

**Pull request closed without merging**